ionics_fits/models/quantum_phys.py

```diff
@@ -98,4 +98,42 @@ def squeezed_state_probs(
     return P_n
 
 
-# pytype: enable=invalid-annotation
+def thermal_displaced_state_probs(
+    n_max: int,
+    n_bar: ModelParameter(lower_bound=0, scale_func=scale_invariant),
+    alpha: ModelParameter(lower_bound=0, scale_func=scale_invariant),
+) -> Array[("num_fock_states",), np.float64]:
+    """Displaced thermal probability distribution.
+
+    For an ion initially in a thermal distribution characterised by an average phonon
+    number n_thermal, we calculate the new probability distribution after applying a
+    Displacement operator D(α).
+
+    formula taken from equation (7) of
+    Ramm, M., Pruttivarasin, T. and Häffner, H., 2014. Energy transport in trapped ion
+    chains. New Journal of Physics, 16(6), p.063062.
+    https://iopscience.iop.org/article/10.1088/1367-2630/16/6/063062/pdf
+
+    :param n_max: the distribution is truncated at a maximum Fock state of ``|n_max>``
+    :param n_bar: the mean Fock state occupation
+    :param alpha: Complex displacement parameter
+    :returns: array of Fock state occupation probabilities
+
+    """
+
+    n_alpha = alpha**2
```

Comment:
maybe `n_bar_alpha` to be more explicit that this is the mean occupation rather than a list of `n`s

Reply:
I am in favour of leaving this extra variable n_bar_alpha, in case in the future we decide to parametrise the function with that directly.